addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -1591,7 +1591,8 @@ export function isUnbreakable(node) {
                 node.getAttribute('t-value') ||
                 node.getAttribute('t-out') ||
                 node.getAttribute('t-raw'))) ||
-        node.classList.contains('oe_unbreakable')
+        node.classList.contains('oe_unbreakable') ||
+        (node.tagName === "A" && node.getAttribute("data-oe-type") === "button")
```

Comment:
We are already checking ```oe_unbreakable``` then why not adding ```oe_unbreakable``` class on Tabs instead of checking for this condition, is there any other cases which we want to handle here, if yes then would you please elaborate which case will handled here?

Reply:
The problem is that in other snippets where the `anchor(<a>) button` is used they all break when the enter event is triggered.
To handle all the snippets had to change `isUnbreakable()`. Of course we can add `oe_unbreakble` to all that snippets individually but the same can be handled in master when we change XML directly.